ogr/services/pagure.py

```diff
@@ -824,6 +824,12 @@ def get_username(self) -> str:
         return_value = self.service.call_api(url=request_url, method="POST", data={})
         return return_value["username"]
 
+    def get_email(self) -> str:
+        request_url = self.service.get_api_url("user", self.get_username())
+
+        return_value = self.service.call_api(url=request_url)["default_email"]
```

Comment:
This addition won't work since Pagure API doesn't support `default_email` parameter in your API call on `GET /api/0/user/<username>`. I just created PR #141 and put your working commits there.